rcljava_common/cmake/Modules/JavaExtra.cmake

```diff
@@ -180,7 +180,7 @@ function(add_source_jar _TARGET_NAME)
 
   cmake_parse_arguments(_add_source_jar
     ""
-    "OUTPUT_NAME"
+    "OUTPUT_NAME;JAR_ROOT_PATH"
```

Comment:
On one hand, it seems like it should be the responsibility of the caller to remove any prefix paths, but on the other hand it's convenient to let the function take care of it. The downside of letting this function remove the prefix is that it is assuming that all source files have the same prefix.

I think it would safer to instead let the caller deal with the prefix and they can inform this function by passing a `WORKING_DIRECTORY` argument used when calling the jar command below.

Please add documentation for the new argument to the docblock above.